### DISCUSSION TITLE
How to get radio button and label inline

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#653] [General]
I created a custom field with radio buttons and put it in a html table. 
Radio buttons and labels are not inline. I tried some CSS but did not get them on the same line. How do i do that?
See my screenprint: https://stage4.xyh6t9w.nl/wp-content/uploads/2024/09/radio-button.png

**Comment by liedekef** [comment]
Based on a screenshot I can't tell that much, but EME forces button and label to be on the same line (using non-breaking space). So if your text shows on the next line, it might be that there's a linebreak  in your label value. Can you demo this live somewhere?

**Comment by JanineKall** [comment]
Hi,You can find it on https://HB-Cafe.nl/evenementen/congres-hoogbegaafdheid-in-werk-en-onderwijs-over-verveling-vervulling-en-verschillen.Op 20 sep 2024 10:37 schreef Franky Van Liedekerke ***@***.***>:
Based on a screenshot I can't tell that much, but EME forces button and label to be on the same line (using non-breaking space). So if your text shows on the next line, it might be that there's a linebreak  in your label value. Can you demo this live somewhere?

**Comment by liedekef** [comment]
your css causes this. In ethos.css there's a definition that causes all labels to be blocks. Since this seems part of your child-theme: either fix that css, or add an extra css part that overrides this ("display: inline" is sufficient)

**Reply by JanineKall** [reply]
Thanks. Tried that before, didn't work so I gave up on that. Tried it again after I read your reply. Took me a while to discover where to put the CSS, but it works now!
